scarb/tests/profiles.rs

```diff
@@ -240,6 +240,33 @@ fn cannot_choose_not_existing_profile() {
         .stdout_matches("error: workspace `[..]` has no profile `custom`\n");
 }
 
+#[test]
+fn can_use_short_profile_args_in_scripts() {
+    let t = TempDir::new().unwrap();
+    ProjectBuilder::start()
+        .name("hello")
+        .manifest_extra(
+            r#"
+            [scripts]
+            build-dev = "scarb --dev build"
+            build-release = "scarb --release build"
+            "#,
+        )
+        .build(&t);
+
+    Scarb::quick_snapbox()
+        .args(["run", "build-dev"])
+        .current_dir(&t)
+        .assert()
+        .success();
+
+    Scarb::quick_snapbox()
+        .args(["run", "build-release"])
+        .current_dir(&t)
+        .assert()
+        .success();
+}
+
```

Comment:
Can we check which profile has been used? 
You can read it from Scarb metadata: https://docs.rs/scarb-metadata/latest/scarb_metadata/struct.Metadata.html#structfield.current_profile

Can we check if cli args take precedence over env var?